tests/models/bloom/test_modeling_flax_bloom.py

```diff
@@ -0,0 +1,281 @@
+# Copyright 2022 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import unittest
+
+import numpy as np  # noqa
+
+from transformers import BloomConfig, BloomTokenizerFast, is_flax_available
+from transformers.testing_utils import is_pt_flax_cross_test, require_flax, require_torch, slow
+from transformers.utils.import_utils import is_torch_available
+
+from ...generation.test_generation_flax_utils import FlaxGenerationTesterMixin
+from ...test_modeling_flax_common import FlaxModelTesterMixin, ids_tensor
+
+
+if is_flax_available():
+    import os
+
+    # The slow tests are often failing with OOM error on GPU
+    # This makes JAX allocate exactly what is needed on demand, and deallocate memory that is no longer needed
+    # but will be slower as stated here https://jax.readthedocs.io/en/latest/gpu_memory_allocation.html
+    os.environ["XLA_PYTHON_CLIENT_ALLOCATOR"] = "platform"
+
+    import jax.numpy as jnp
+    from transformers import FlaxBloomForCausalLM, FlaxBloomModel
+
+if is_flax_available() and is_torch_available():
+    from transformers.models.bloom.modeling_bloom import build_alibi_tensor
+    from transformers.models.bloom.modeling_flax_bloom import build_alibi_tensor_flax
+
+
+def prepare_opt_inputs_dict(config, input_ids, attention_mask=None, head_mask=None):
```

Comment:
Head mask not used
```suggestion
def prepare_bloom_inputs_dict(config, input_ids, attention_mask=None):
```

Reply:
Thanks for the catch!

Reply:
Also changed the function name in a test